GOVERNANCE.md

```diff
@@ -1,25 +1,100 @@
+
+---
+# Maintainers
+New maintainers must be nominated by an existing maintainer and must be elected by a supermajority of existing maintainers. Likewise, maintainers can be removed by a supermajority of the existing maintainers or can resign by notifying one of the maintainers.
+
+---
+# Supermajority
+A supermajority is defined as two-thirds of members in the group. A supermajority of Maintainers is required for certain decisions as outlined above. A supermajority vote is equivalent to the number of votes in favor of being at least twice the number of votes against. For example, if you have 5 maintainers, a supermajority vote is 4 votes. Voting on decisions can happen on the mailing list, GitHub, Slack, email, or via a voting service, when appropriate. Maintainers can either vote "agree, yes, +1", "disagree, no, -1", or "abstain". A vote passes when supermajority is met. An abstain vote equals not voting at all.
+
+---
+# Decision Making
+Ideally, all project decisions are resolved by consensus. If impossible, any maintainer may call a vote. Unless otherwise specified in this document, any vote will be decided by a supermajority of maintainers.
+
+Votes by maintainers belonging to the same company will count as one vote; e.g., 4 maintainers employed by fictional company Valerium will only have one combined vote. If voting members from a given company do not agree, the company's vote is determined by a supermajority of voters from that company. If no supermajority is achieved, the company is considered to have abstained.
+
+---
+# Proposal Process
+One of the most important aspects in any open source community is the concept of proposals. Large changes to the codebase and / or new features should be preceded by a proposal in our carvel repo. This process allows for all members of the community to weigh in on the concept (including the technical details), share their comments and ideas, and offer to help. It also ensures that members are not duplicating work or inadvertently stepping on toes by making large conflicting changes.
+
+The [project roadmap](https://github.com/vmware-tanzu/carvel/blob/develop/ROADMAP.md) is defined by accepted proposals.
+
+Proposals should cover the high-level objectives, use cases, and technical recommendations on how to implement. In general, the community member(s) interested in implementing the proposal should be either deeply engaged in the proposal process or be an author of the proposal.
+
+The proposal process including a [Proposal Template](https://github.com/vmware-tanzu/carvel/tree/develop/proposals#proposal-template) is covered at length within the [proposal directory](https://github.com/vmware-tanzu/carvel/tree/develop/proposals).
+
+---
+# Proposal Lifecycle
```

Comment:
I would remove this section to dedupe. If we have a single source of truth (i.e. the proposal dir) then it will be easier to maintain.